src/main/java/com/google/sps/data/Player.java

```diff
@@ -6,6 +6,7 @@
   private String id;
   private String imageID;
   private String currentPageID;
+  private int experience;
```

Comment:
maybe name this something like "currentExperiencePoints" and add a doc describing what it is and how it works. It's not as "standard" as the other fields in this file and so probably deserves a comment.